front-end/src/components/AppNavBar.tsx

```diff
@@ -73,30 +80,21 @@ export default function AppNavBar(props: Props) {
 
   return (
     <div>
+      {props.loggedInUser ?
+      
       <Navbar light expand="md">
         <NavbarToggler onClick={toggle} />
         <Collapse isOpen={isOpen} navbar>
           <Nav vertical className="ml-auto" navbar>
-            <NavLink to="/" className="navbar-brand font-title">
-              🗑🔥
-            </NavLink>
-            <NavItem onClick={close}>
-              <NavLink to="/" className="nav-link">
-              {Icons.homeIcon} Feed
+          <NavLink to="/" className="navbar-brand font-title text-center">
+                {Icons.tamagoEgg("45", "45")}
               </NavLink>
-            </NavItem>
-            {props.loggedInUser ? (
-              loggedIn()
-            ) : (
-              <NavItem onClick={close}>
-                <NavLink to="/auth/" className="nav-link">
-                {Icons.loginIcon} Log In
-                </NavLink>
-              </NavItem>
-            )}
+            {loggedIn()}
           </Nav>
         </Collapse>
       </Navbar>
+      : <h1 style={{padding: '4rem'}}>{Icons.tamagoEgg("150", "150")}</h1>
```

Comment:
Do we have a way to navigate to the login page if we are not logged in?

Reply:
The changes I made always redirect to the login page if you're not logged in. Otherwise if you try to navigate anywhere other than the login page you'll get a 404

Reply:
Thanks! While there's minor UX trade-offs with this approach (can't press the back button while in the login screen), I think this implementation should be good for our project :+1:

Reply:
maybe I misunderstood but you can click the egg to go back to the login screen? Like if you get a 404, there's a button to go to login and you could also click the egg

Reply:
wait I forgot to push the change I made so you can click the egg to go back to the login screen lol I'll do that now

Reply:
If I get redirected to the login screen, I can't press the back button, as I get sent back to the page that will redirect me back to the login screen. React router handles these kinds of cases with it's auth features, but we don't need to worry about it for this project